.github/scripts/validate-semver.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -euo pipefail
+
+sem_ver_pattern="^[vV](0|[1-9][0-9]*)\\.(0|[1-9][0-9]*)\\.(0|[1-9][0-9]*)(\\-[0-9A-Za-z-]+(\\.[0-9A-Za-z-]+)*)?(\\+[0-9A-Za-z-]+(\\.[0-9A-Za-z-]+)*)?$"
+
+die () {
+    echo >&2 "$@"
+    exit 1
+}
```

Comment:
why we have mixed usage of die here and exit 1 other places?

Reply:
so I had reused the script used in istio-workspaces: https://github.com/maistra/istio-workspace/blob/master/scripts/release/validate.sh. I can use a simple echo + exit 1 in the validate-semver.sh if that's okay.